schema.sql

```diff
@@ -0,0 +1,287 @@
+    created_at TIMESTAMP NOT NULL,
+    message TEXT,
+    resolved BOOL NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (challenge_id) REFERENCES pot_payout_challenge (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+-- Table donation
+CREATE TABLE
+  donation (
+    id SERIAL PRIMARY KEY,
+    donor_id VARCHAR NOT NULL,
+    total_amount VARCHAR NOT NULL,
+    total_amount_usd DECIMAL(10, 2),
+    net_amount VARCHAR NOT NULL,
+    net_amount_usd DECIMAL(10, 2),
+    ft_id VARCHAR,
+    pot_id INT,
+    matching_pool BOOLEAN NOT NULL,
+    message TEXT,
+    donated_at TIMESTAMP NOT NULL,
+    recipient_id VARCHAR,
+    protocol_fee VARCHAR NOT NULL,
+    protocol_fee_usd DECIMAL(10, 2),
+    referrer_id VARCHAR,
+    referrer_fee VARCHAR,
+    referrer_fee_usd DECIMAL(10, 2),
+    chef_id VARCHAR,
+    chef_fee VARCHAR,
+    chef_fee_usd DECIMAL(10, 2),
+    tx_hash VARCHAR,
+    FOREIGN KEY (donor_id) REFERENCES account (id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (recipient_id) REFERENCES account (id)
+  );
+
+-- Table activity
+CREATE TABLE
+  activity (
+    id SERIAL PRIMARY KEY,
+    signer_id VARCHAR NOT NULL,
+    receiver_id VARCHAR NOT NULL,
+    timestamp TIMESTAMP NOT NULL,
+    action_result VARCHAR,
+    tx_hash VARCHAR,
+    type
+      ENUM(
+        'Donate_Direct',
+        'Donate_Pot_Public',
+        'Donate_Pot_Matching_Pool',
+        'Register',
+        'Deploy_Pot',
+        'Process_Payouts',
+        'Challenge_Payout',
+        'Submit_Application',
+        'Update_Pot_Config',
+        'Add_List_Admin',
+        'Remove_List_Admin'
+      ) NOT NULL
+  );
+
+CREATE TABLE
+  pot_admin (
+    pot_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_id, admin_id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+-- pot index
+
+-- CREATE INDEX "deploy_time_idx" ON pot (deployed_at);
+CREATE INDEX idx_pot_application_start ON pot (application_start);
+CREATE INDEX idx_pot_application_end ON pot (application_end);
+CREATE INDEX "idx_pot_deployer_id" ON pot (deployer_id);
+
+-- pot application index
+
+CREATE INDEX idx_pot_application_pot_id ON pot_application (pot_id);
+CREATE INDEX idx_pot_application_applicant_id ON pot_application (applicant_id); 
+CREATE INDEX idx_pot_application_submitted_at ON pot_application (submitted_at);
+
+
+-- payout index
+CREATE INDEX idx_pot_payout_recipient_id ON pot_payout (recipient_id);
+
+-- donation index
+CREATE INDEX idx_donation_donor_id ON donation (donor_id);
+CREATE INDEX idx_donation_pot_id ON donation (pot_id);
+CREATE INDEX idx_donation_donated_at ON donation (donated_at);
+
+-- activity index
+CREATE INDEX idx_activity_timestamp ON activity (timestamp);
+
+-- CREATE INDEX idx_pot_payout_recipient_id ON pot_payout (recipient_id);
+-- CREATE INDEX idx_pot_payout_ft_id ON pot_payout (ft_id);
+CREATE INDEX idx_pot_payout_paid_at ON pot_payout (paid_at);
+
+
```

Comment:
add:

indexes for account (`total_donations_received_usd`, `total_matching_pool_allocations_usd`, `total_donated_usd`, `donors_count` - we will almost certainly want to query ordered results based on these columns)

indexes for list on `created_at` and `updated_at`

composite index for list registration/status: `CREATE INDEX idx_list_id_status ON list_registration(list_id, status);`